project_admin/tests/tests.py

```diff
@@ -43,3 +43,174 @@ def test_admin_login_fail(self):
         response = c.post("/project-admin/login/", {'password': 'meep'})
         self.assertEqual(response.status_code, 200)
         self.assertContains(response, "Password incorrect")
+
+    def test_config_homepage_text_logged_out(self):
+        """
+        Test config_homepage_text function when logged out
+        """
+        c = Client()
+        response = c.get("/project-admin/config-homepage-text/")
+        self.assertEqual(response.status_code, 302)
```

Comment:
Please use `self.assertRedirects` to check for redirects. You can use it like this: `self.assertRedirects(response, '/where-does-it-redirect-to/')`